test/e2e-go/features/transactions/proof_test.go

```diff
@@ -126,5 +126,9 @@ func TestTxnMerkleProof(t *testing.T) {
 
 	elems[proofresp.Idx] = &element
 	err = merklearray.Verify(blk.TxnRoot.ToSlice(), elems, &proof)
-	a.NoError(err)
+	if err != nil {
+		t.Logf("blk.TxnRoot : %v \nproof path %v \ndepth: %d \nStibhash %v", blk.TxnRoot.ToSlice(), proof.Path, proof.TreeDepth, proofresp.Stibhash)
+		t.FailNow()
```

Comment:
I prefer to still call `a.NoError(err)` here instead of `t.FailNow()`. 
`t.FailNow()` is not commonly used, and might act funny when using other objects implementing the interface.